iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/KubernetesParser.java

```diff
@@ -88,6 +89,13 @@ private FileTree evaluateAndParseHelmFile(String source, InputFileContext inputF
     return super.parse(evaluatedAndCleanedSource, inputFileContext, FileTree.Template.HELM);
   }
 
+  private static void readLinesSizes(String source, InputFileContext ctx, LocationShifter locationShifter) {
```

Comment:
Maybe it can be a member method of `LocationShifter`? It seems to me, we put a lot of pieces of Helm-related logic into `KubernetesParser`